src/module.c

```diff
@@ -417,7 +417,8 @@ static int internalAdd(RedisModuleCtx *ctx,
     if (timestamp <= series->lastTimestamp && series->totalSamples != 0) {
         if (SeriesUpsertSample(series, timestamp, value, dp_override) != REDISMODULE_OK) {
             RTS_ReplyGeneralError(ctx,
-                                  "TSDB: Error at upsert, update is not supported in BLOCK mode");
+                                  "TSDB: Error at upsert, update is not supported when "
+                                  "DUPLICATE_POLICY in set to BLOCK mode");
```

Comment:
I believe there is a typo here. It should be "DUPLICATE_POLICY **is** set to BLOCK mode", right?

Reply:
you're absolutely right. Fixing it tomorrow morning :+1:

Reply:
@amotl thank you for noticing it. We're merged the typo fix: https://github.com/RedisTimeSeries/RedisTimeSeries/pull/828